lib/rules/string-quotes/__tests__/index.js

```diff
@@ -40,6 +40,10 @@ testRule(rule, {
     {
       code: 'a { /* "horse" */ }',
       description: "ignores comment"
+    },
+    {
+      code: '@charset "utf-8"',
+      description: "ignore @charset rules"
     }
```

Comment:
Could you add this same test to the "double quotes" test around line #119 as a safe guard of sorts please

Reply:
Added below ...

Reply:
Thanks, though you added it to the [SCSS tests](https://github.com/stylelint/stylelint/blob/6d89566dca654f990ff26d4a11d44cb9b5edee58/lib/rules/string-quotes/__tests__/index.js#L160-L163) at lines 160-163, which won't hurt either, can you add it after line 118 please :)

Reply:
Whoops. Yes added there.